app/helpers/activity_helper.rb

```diff
@@ -17,4 +17,8 @@ def link_to_activity_parent(parent:, user:)
     return parent.title if parent.fund? && user.delivery_partner?
     link_to parent.title, organisation_activity_path(parent.organisation, parent), {class: "govuk-link govuk-link--no-visited-state"}
   end
+
+  def custom_capitalisation(level)
```

Comment:
I'd like to see a spec for this in `activity_helper_spec` to make it obvious what it's doing

Reply:
That's true. I forgot about doing one. Thanks!